src/schema/schema.graphql

```diff
@@ -2016,6 +2016,7 @@ input DocumentInput {
 input AttachmentInput {
 	attachmentId: String
 	documents: [DocumentInput]
+	messages: [DocumentInput]
```

Comment:
should we rename DocumentInput in this case as we are using for both document and message

Reply:
or create separate entity for MessageInput

Reply:
Done.